balrogscript/src/balrogscript/submitter/cli.py

```diff
@@ -261,6 +306,81 @@ def update_latest():
 
         retry(update_latest, sleeptime=2, max_sleeptime=2, attempts=10)
 
+    def run_backend2(
+        self, platform, buildID, productName, branch, appVersion, locale, hashFunction, extVersion, schemaVersion, isOSUpdate=None, **updateKwargs
+    ):
+        log.info("Using backend version 2...")
+        requests_api = get_balrog_api(auth0_secrets=self.auth0_secrets)
+        headers = {"Accept-Encoding": "application/json", "Accept": "application/json", "Content-Type": "application/json", "Referer": self.api_root}
+
+        def do_request(url, method="get", data=None):
+            logging.debug("Balrog request to %s via %s", url, method.upper())
```

Comment:
Rather than doing all of this manual logging ourselves I wonder if we should increase the `requests` or `urllib3` verbosity? Up to you if you want to try it.

Reply:
I mostly copied the manual logging from the upstream api we used in v1 for this. I didn't feel like playing with requests/urllib3 verbosity here at this point.